.azure-pipelines/template-skipvstest.yml

```diff
@@ -0,0 +1,19 @@
+steps:
+- script: |
```

Comment:
We can exclude the script if the Build.Reason is not "PullRequest", right?
```
steps:
- ${{ eq(variables["Build.Reason"], "PullRequest") }}:
  - script:
     ....  
 ```

If not work, another option:
```
steps:
- script: |
    ....
  condition: eq($(Build.Reason)", "PullRequest")
```